### ISSUE TITLE
NTR: Ear: tympanic membrane thick (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7811]
New term request Ear: tympanic membrane thick (elthat_150504173754): Thickening; Thickened  UMLS: 0395854

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000183
- label: Thickened tympanic membrane
- parents: [Abnormality of the tympanic membrane (HP:0040090)](https://hpo.jax.org/app/browse/term/HP:0040090)
- pmid: PMID:28846242|PMID:23673509
- def: The tympanic membrane (eardrum, myringa) is a thin, semitransparent, oval membrane, approximately 1 cm in diameter, that separates the external acoustic meatus from the tympanic cavity. Thickening refers to an increase of the dimension across the membrane.
- comment: See Figure 2 of PMID:23673509.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7811
- synonyms:
